domain-model/src/test/java/uk/gov/hmcts/cmc/domain/amount/TotalAmountCalculatorTest.java

```diff
@@ -109,10 +109,82 @@ public void totalTillTodayShouldReturnCalculatedTotalAmountPlusInterestWhenCalcu
                     .build()
             ).build();
 
-        assertThat(TotalAmountCalculator.totalTillDateOfIssue(claimNoInterest))
+        assertThat(TotalAmountCalculator.totalTillDateOfIssue(claimStandardInterest))
             .isEqualTo(Optional.of(format(new BigDecimal("60"))));
     }
 
+    @Test
+    public void totalTillTodayWhenClaimIssuedOnIsToday() {
```

Comment:
I would love rename the test case to say what is expected e.g. `totalTillTodayShouldNotIncludeInterestWhenClaimIssuedOnIsToday`.